configs/ensemble/2021.02.20.ini

```diff
@@ -0,0 +1,12 @@
+[DEFAULT]
```

Comment:
we love configs. is this the only config?

Reply:
it is for the ensemble model as this PR is the one that introduces them. But we have them for creating the training data and cluster analysis too